certora/harnesses/AccessManagerHarness.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import "../patched/access/manager/AccessManager.sol";
+
+contract AccessManagerHarness is AccessManager {
+    constructor(address initialAdmin) AccessManager(initialAdmin) {}
+
+    // FV
+    function canCall_1(address caller, address target, bytes4 selector) external view returns (bool result) {
```

Comment:
I'd name these as `canCall_immediate` instead. I think it's quite difficult to follow the spec without known exactly which value is

Reply:
Agree with this. @Amxx was there an objection to this rename?

Reply:
done

Reply:
Thanks @Amxx !